schemas/cloudToDevice/ground_fix/ground-fix.json

```diff
@@ -25,9 +25,29 @@
              },
              "fulfilledWith":{
                "$ref": "#/definitions/FulfilledWith"
+             },
+             "meta":{
+               "type": "object",
+               "properties": {
+                  "anchors": {
+                     "type": "array",
+                     "items": {
+                        "type": "object",
+                        "properties": {
+                           "macAddress": { "type": "string", "pattern": "^((([0-9A-F]{2}:){5})|([0-9A-F]{10}))([0-9A-F]{2})$" },
+                           "name": { "type": "string" }
+                        },
+                        "required": ["macAddress"],
```

Comment:
```
"anchors": [{ 
                "macAddress": "9F:11:7D:3A:4F:FF"
            }]
 ```
 what about the "name" of the anchor?  the example use case such as "warehouse 1".
 is this intended to be send back with the mac address?

Reply:
also can "macAddress" be just "MAC"?

or do we need to indicate that it is a MAC?  are there other anchor sources?
could this array be in [ anchor_name, anchor_source ] format (or something similar)?

Reply:
I think we should plan on adding GNSS at some point, as I mentioned in the call this morning.  This would entail defining a lat/lng and radius (accuracy) for the anchor point.

We can discuss trading off the costs and benefits of these approaches to detect when a device is at an anchor point:
1. Whenever the device sends a PVT or NMEA report to the cloud, it includes lat/lng and accuracy in meters. The cloud would process the stored list of anchor points and check whether the device is likely to be within the radius of one of them, and if so, trigger whatever our anchor point behavior is.
--> disadvantage: AWS costs
2. The cloud sends to the device a list of anchor points, which it stores in flash.  Whether it is connected to a network or not, when the device gets a GNSS fix, it computes whether it is in close proximity to any anchor point.  If so, it connects to the cloud and sends a message of some sort to the cloud, which triggers anchor point behavior.
--> disadvantage: battery consumption, firmware complexity

Reply:
`The cloud would process the stored list of anchor points and check whether the device is likely to be within the radius of one of them, and if so, trigger whatever our anchor point behavior is.`

this seems like a new type of behavior/endpoint.
for GNSS messages, there is no response back to the device, it is just a device message.

Reply:
There is an idea for a future feature that would allow the device to prevent subsequent cloud requests if it knew the macAddresses of its anchors. Next time it scanned, if the macAddress of a known anchor is detected, it could prevent firing up the radios for another cloud request. The macAddress is included for this use case. 

The array format would be difficult because name is an optional parameter when creating an anchor, so not all anchors will have them.

Reply:
oh, "name" is in the schema already 🤦 
i was looking for it in the examples.

yes, i was just asking about other potential non-MAC anchor sources... sounds like GNSS could be one.
the "future feature" sounds like it could be implemented now in firmware.

Reply:
@plskeggs  This sounds like a great way to leverage anchors. I like the idea in option #1 better, as AWS cloud costs are nominal compared to the extra battery usage / FW compexity in option #2. Plus it would be centralized on the cloud and all devices would get it without upgrading FW. 

What would the cloud do if a nearby anchor is detected? The most useful thing I can think of is sending out an alert. Was there something else you had in mind?